src/components/SideNavigation/SideNavigation.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
+import './SideNavigation.scss';
+import { useRef } from 'react';
+import EventsPage from '../EventsPage/EventsPage';
+import HomePage from '../HomePage/HomePage';
+import AboutPage from '../AboutPage/AboutPage';
+
+//NOTE: For Mobile View style - Have to refresh screen
+export default function SideNavigation() {
+  const eventsPageRef = useRef(null);
+  const homePageRef = useRef(null);
+  const aboutPageRef = useRef(null);
+
+  const sections = [
```

Comment:
@Hariharan482 Kindly move this to constants

Reply:
moved it to constants and created a util to create a ref and assign it to each section